File: assignment 1/python/programs/9.py
# Write a program to find frequency of a given number ‘k’
arr = []
size = int(input("Enter the size of the array: "))
for i in range(size):
    element = int(input(f"Enter element {i+1}: "))
    arr.append(element)
k=int(input("enter the number:"))
count=arr.count(k)
print(count)

File: assignment 1/c/programs/9.c
#include <stdio.h>

int findFrequency(int array[], int size, int k)
{
    int frequency = 0;

    for (int i = 0; i < size; i++)
    {
        if (array[i] == k)
        {
            frequency++;
        }
    }

    return frequency;
}

int main()
{
    int array[100];
    int size;
    int k;

    printf("Enter the size of the array: ");
    scanf("%d", &size);

    printf("Enter the elements of the array: ");
    for (int i = 0; i < size; i++)
    {
        scanf("%d", &array[i]);
    }

    printf("Array: ");
    for (int i = 0; i < size; i++)
    {
        printf("%d ", array[i]);
    }
    printf("\n");

    printf("Enter the number to find its frequency: ");
    scanf("%d", &k);

    int frequency = findFrequency(array, size, k);
    printf("Frequency of %d in the array: %d\n", k, frequency);

    return 0;
}